src/wmnds/assets/sass/_vars.scss

```diff
@@ -18,7 +18,8 @@ $palettes: (
   information: #84329b,
   disable: #676869,
   plannedDisruption: #ffdd00,
-  background: #f3f2f1
+  background: #f3f2f1,
+  emergency: #231f20
```

Comment:
I think this is the same colour as the "text" var.
Just update to use the text colour instead and remove this one.